stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -5554,3 +5554,17 @@ func.func @invalid_dimension_attr(%arg0: tensor<?x?xf32, #stablehlo.type_extensi
   %result = "stablehlo.set_dimension_size"(%arg0, %arg1) {dimension = -1 : i64} : (tensor<?x?xf32, #stablehlo.type_extensions<bounds = [3, -1]>>, tensor<i32>) -> tensor<*xf32>
   func.return %result : tensor<*xf32>
 }
+
+// -----
+
```

Comment:
Looks like this files convention for positive test cases is `CHECK LABEL`. Same feedback for f8E5M2 below.